### PR TITLE
R1.0 to master

### PR DESCRIPTION
**Pull request by av8ramit**
Merging r1.0 back into master.

### CODE REVIEW

**Comment by gunan** [comment]
@cwhipkey @jhseu Could you take another look at this PR?

File: tensorflow/tools/ci_build/Dockerfile.gpu [hunk 1]
```diff
@@ -1,7 +1,12 @@
-FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04
+FROM nvidia/cuda:8.0-cudnn5-devel-ubuntu14.04
```
Comment (jhseu) [inline tensorflow/tools/ci_build/Dockerfile.gpu]: Exclude this file? It won't build for the reasons I mentioned in the other change.

Comment (av8ramit) [inline tensorflow/tools/ci_build/Dockerfile.gpu]: @gunan:
"As I mentioned before, this governs the package we build for releases.
We cannot build 1.1 release on 16.04 -> so this has to be merged.
Could you point Amit to the changes that need to be done, so we can add them to this PR?"

Comment (gunan) [inline tensorflow/tools/ci_build/Dockerfile.gpu]: OK, we discussed with @jhseu offline.
It looks like somehow the merge back does not capture all the changes that need to be done to get us back to ubuntu 14.04
Once we do the merge back, he will roll back the change to upgrade our dockerfiles to ubuntu 16.

So let's leave out this file for now as @jhseu suggested.

**Review by jhseu — APPROVED** [review]

**Comment by av8ramit** [comment]
@gunan any other issues before I merge? CLA?

**Comment by gunan** [comment]
CLA bot does this during merge PRs. All CLAs are signed, thats all we need.

**Review by gunan — APPROVED** [review]

**Pull request merged**